### DISCUSSION TITLE
Troubleshooting steps if DS3 rumble only works with Xinput but not other modes?

### DISCUSSION THREAD

**Discussion by TrantaLocked** [nefarius/DsHidMini#290] [Support]
I'm playing Metal Gear Rising Revengeance with a DS3 and would like to use one of the driver modes that populate Steam Big Picture Mode with DualShock icons and images instead of Xbox ones. The problem is that - with or without Steam Input enabled, wired or bluetooth - the only driver mode I can actually get rumble working with in this game is Xinput, which populates with Xbox icons. And there's no input at all if I use a non-Xinput driver mode without Steam Input.

Ideally I would want to use SXS/RPCS3 mode with Steam Input with full rumble. Is that achievable?

**Comment by Kanuan** [comment]
1. Change to DS4Windows mode
2. Use Ryochan7's DS4Windows
    - https://vigem.org/projects/DsHidMini/DS4-Mode-User-Guide/
4. Configure a profile set for DualShock 4 emulation
5. Make sure DS4Windows is running under a custom .exe name so Steam doesn't detect it and ignores virtual DS4 controllers
    - though I think Steam doesn't do this anymore, so... just set the DS4Windows profile to DS4 emulation
